### PR TITLE
Added high level table

### PR DESCRIPTION
**Pull request by claire-melbourne**
Modified the table to use markdown (so no colspan option) and added "A", "B", etc. to differentiate between different environments, services etc.
Screen Shot 2023-01-06 at 4 41 49 PM

### CODE REVIEW

**Comment by flybayer** [comment]
Looking at how this is displayed in https://claire-melbourne-table-using-code.docs-f42607od.preflig.ht/docs/guides/config/using-code and I feel it's just too confusing.

To be better than the previous bullet list, I think it needs to be a graphic where we can have boxes and nested boxes to indicate the different groupings and hierarchy. OR instead of nested, a tree structure would also work.

**Comment by claire-melbourne** [comment]
@flybayer Totally. How about simplifying it a little and using something like this?
```mermaid
graph TD;
    flightcontrol.json-->Project;
    Project-->Environment_A;
   Project-->Environment_B;
    Environment_A-->Service_1;
   Environment_A-->Service_2;
```

**Comment by flybayer** [comment]
@claire-melbourne perfect!

File: motif-docs/pages/guides/config/using-code.mdx [hunk 1]
```diff
@@ -13,12 +13,21 @@ import dashBoardPic from '../../../../public/dashboard-maintenance-window.png'
...
+- An `Environment` can have multiple `Service`'s
+
+```mermaid
+graph TD;
+    flightcontrol.json-->Project;
+    Project-->Environment_A;
+   Project-->Environment_B;
+    Environment_A-->Service_1;
+   Environment_A-->Service_2;
+```
```
Comment (flybayer) [inline motif-docs/pages/guides/config/using-code.mdx:28]: Not sure if you checked the preview deployment, but it seems we don't currently support mermaid.

image

I'll just close this PR for now until we add mermaid support

**Pull request closed without merging**